lib/vet360/models/email.rb

```diff
@@ -18,6 +18,13 @@ class Email < Base
         format: { with: EVSS::PCIU::EmailAddress::VALID_EMAIL_REGEX },
         length: { maximum: 255, minimum: 6 }
       )
+
+      def details
+        {
+          id: id,
+          email_address: email_address
+        }
```

Comment:
@bshyong @ncksllvn - anything else you'd like to see returned for an email?

Reply:
💯

Reply:
The only thing I can think is potentially an `updatedAt` might be useful, in case UX decides we need to draw attention to when a field updates.